soh/soh/Enhancements/randomizer/3drando/playthrough.cpp

```diff
@@ -58,10 +59,10 @@ int Playthrough_Init(uint32_t seed, std::unordered_map<RandomizerSettingKey, uin
     GenerateHash();
     WriteIngameSpoilerLog();
 
-    if (Settings::GenerateSpoilerLog) {
+    if (/*Settings::GenerateSpoilerLog TODO: do we ever not want to write a spoiler log?*/ true) {
```

Comment:
Race seeds?

Reply:
tl;dr I agree, but actually implementing that option involves some work that is out of scope for this PR.

Yeah I remember your comment on the other PR. I just haven't gone through and implemented that option in the new system yet, felt out of scope here and would be done in a cleanup PR of some kind soon. There's also some other changes involved in that, for example with race seeds I think we would still want to generate a json file, it would just only have the settings and the seed string. That way we can drag and drop to gen the same seed for races without sharing the entire spoiler log.